api/sip/client.py

```diff
@@ -252,127 +248,94 @@ def __init__(self, target_server, target_port, login_user_id=None,
             escaped = self.separator
         self.separator_re = re.compile(escaped + "([A-Z][A-Z])")
 
+        self.sequence_number = 0
```

Comment:
I don't think we need sequence numbers at all since a given connection is never sending more than one message. (Resent messages don't count because you're not supposed to increment the sequence number.)